packages/core/src/external/fhir/shared/age.ts

```diff
@@ -0,0 +1,43 @@
+import { Age } from "@medplum/fhirtypes";
+import { buildDayjs } from "@metriport/shared/common/date";
+import dayjs from "dayjs";
+
+export function addAgeToDob(age: Age | undefined, dob: string | undefined): string | undefined {
+  if (!dob) return undefined;
+  if (!age?.value) return undefined;
+  const dayjsUnit = ageUnitToDayjsUnit(age.unit);
+  if (!dayjsUnit) return undefined; // could try to infer based on the DOB + current date
+  return buildDayjs(dob).add(age.value, dayjsUnit).toISOString();
+}
+
+function ageUnitToDayjsUnit(unit: string | undefined): dayjs.ManipulateType | undefined {
```

Comment:
https://download.hl7.de/documents/ucum/ucumdata.html

add as a docstring so people know where the abbreviations come from?

Reply:
Done